source/getting-started/managing-dependencies.md

```diff
@@ -0,0 +1,141 @@
+Ember CLI uses the [Bower](http://bower.io) package manager, making it easy
+to keep your front-end dependencies up to date. NPM is used for development
+dependencies (like addons).
+
+## Bower Configuration
+
+The Bower configuration file, `bower.json`, is located at the root of your Ember
+CLI project, and lists the dependencies for your project. Executing `bower install`
+will install all of the dependencies listed in `bower.json` in one step.
+
+Ember CLI is configured to have git ignore your `bower_components` directory by
+default. Using the Bower configuration file allows collaborators to fork your
+repo and get their dependencies installed locally by executing
+`bower install` themselves.
```

Comment:
I think this paragraph can be cut out entirely. `bower install` is covered above, and the rest isn't really necessary to know.